{{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py

```diff
@@ -1,23 +1,22 @@
-from anyblok_pyramid.tests.testcase import PyramidBlokTestCase
 
 
-class TestCanigooRestApi(PyramidBlokTestCase):
+class TestCanigooRestApi:
```

Comment:
Maybe, you should add fixture rollback_registry on the class be sur that

1. registry exist
2. registry is rollbacked between each test

Reply:
oh yes you're right !

Reply:
@jssuzanne, I'm not that aware about pytest is the [class decorator](https://github.com/AnyBlok/cookiecutter-anyblok-project/pull/23/commits/2aa78531fc82e6158bbc58ef89c2b1991166be9f) enough ?

Reply:
it looks like :) https://docs.pytest.org/en/2.8.7/fixture.html#using-fixtures-from-classes-modules-or-projects